frontend/src/schema.graphql

```diff
@@ -183,8 +155,37 @@ type RoleInfo {
   large: Boolean!
 }
 
+"Represents an Opencast series."
+type Series {
+  id: ID!
+  opencastId: String!
+  title: String!
+  created: DateTimeUtc
+  metadata: ExtraMetadata
+  syncedData: SyncedSeriesData
+  hostRealms: [Realm!]!
+  events: [AuthorizedEvent!]!
+  """
+    Returns `true` if the realm has a series block with this series.
+    Otherwise, `false` is returned.
+  """
+  isReferencedByRealm(path: String!): Boolean!
```

Comment:
I noticed: this endpoint still needs to be changed to also check for playlist blocks. Currently, having a playlist blocks on a page, and then clicking on videos, will always redirect to the direct link as the video does not count as being included in the realm. And the docs of this endpoint need to be adjusted as well.